scripts/lib/install

```diff
@@ -151,7 +151,6 @@ read -r -a ADDITIONAL_PACKAGES <<<"${ADDITIONAL_PACKAGES:-}"
 # zulip::profile::docker for Docker.  Use
 # e.g. zulip::profile::app_frontend for a Zulip frontend server.
 PUPPET_CLASSES="${PUPPET_CLASSES:-zulip::profile::standalone}"
-VIRTUALENV_NEEDED="${VIRTUALENV_NEEDED:-yes}"
```

Comment:
This seems unrelated to not installing `node`?

Reply:
Ahh it's just in the next commit, fixing.

Reply:
Thanks!